velox/functions/prestosql/tests/MapEntriesTest.cpp

```diff
@@ -117,3 +119,33 @@ TEST_F(MapEntriesTest, constant) {
       })));
   test::assertEqualVectors(expected, result);
 }
+
+TEST_F(MapEntriesTest, outputSizeIsBoundBySelectedRows) {
+  auto function =
+      exec::getVectorFunction("map_entries", {MAP(BIGINT(), BIGINT())}, {});
+
+  auto map = makeMapVector<int64_t, int64_t>(
+      100,
+      [](vector_size_t row) { return row % 5; },
+      [](vector_size_t row) { return row % 7; },
+      [](vector_size_t row) { return row % 11; });
+  std::vector<VectorPtr> inputsVector = {map};
+
+  // Only the first 5 rows selected.
+  SelectivityVector rows(5);
+  // This is larger than input map size but, the new rows are not selected.
```

Comment:
Instead of "new rows", maybe say "rows beyond the input vector size"?

Reply:
done